sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSampleDao.java

```diff
@@ -343,7 +343,7 @@ public boolean aliasExists(String alias) throws IOException {
     query.setLong("id", projectId);
     @SuppressWarnings("unchecked")
     List<Sample> records = query.list();
-    return fetchSqlStore(records);
+    return records;
```

Comment:
Yes. The only time this `listByProjectId` method is currently called is to list the Samples for the Edit Project page, and that doesn't require the full loading of all child entities for all samples. I can add a comment to that effect on the method to clarify in case someone wants to use the method in the future.